doc/rbd/rbd-snapshot.rst

```diff
@@ -120,13 +122,13 @@ For example::
 
 
 .. note:: Ceph OSDs delete data asynchronously, so deleting a snapshot 
-   doesn't free up the disk space immediately.
+   doesn't immediately free up the underlying OSDs' capacity.
```

Comment:
Plural possessive is not an aspect of the English language that we often see deployed in technical writing with such command and confidence. dulce est videre suos

Reply:
:D